analysis_options.yaml

```diff
@@ -1 +1,14 @@
-include: package:pedantic/analysis_options.yaml
+include: package:flutter_lints/flutter.yaml
+
+analyzer:
+  exclude: [ lib/src/api/proto/**/*.dart,
```

Comment:
Suggest using yml list
```yaml
analyzer:
  exclude:
      - lib/src/api/proto/**/*.dart
      -  ...
```

Reply:
addressed in https://github.com/obrunsmann/flutter_nakama/pull/26/commits/efd1d3bfba26e403efc2c3236702a3b06b4ab635